packages/osd-plugin-helpers/README.md

```diff
@@ -37,37 +38,44 @@ $ plugin-helpers help
   Usage: plugin-helpers [command] [options]
 
   Commands:
-      build
-        Copies files from the source into a zip archive that can be distributed for
-        installation into production OpenSearch Dashboards installs. The archive includes the non-
-        development npm dependencies and builds itself using raw files in the source
-        directory so make sure they are clean/up to date. The resulting archive can
-        be found at:
+    build
+      Copies files from the source into a zip archive that can be distributed for
+      installation into production OpenSearch Dashboards installs. The archive includes the non-
+      development npm dependencies and builds itself using raw files in the source
+      directory so make sure they are clean/up to date. The resulting archive can
+      be found at:
+
+        build/{plugin.id}-{opensearchDashboardsVersion}.zip
+
+      Options:
+        --skip-archive                       Don't create the zip file, just create the build/opensearch-dashboards directory
+        --opensearch-dashboards-version, -v  OpenSearch Dashboards version that the
```

Comment:
is it incomplete help text?

Reply:
It is the output of the CLI `—help`

Reply:
Yeah, this was the existing help text, but we should go ahead and update it to complete the sentence - something like:

```suggestion
        --opensearch-dashboards-version, -v  OpenSearch Dashboards version that the built plugin will target
```